Makefile

```diff
@@ -1,9 +1,15 @@
-all:
+all: dir
 	docker compose -f docker-compose.yml up --build
 build:
 	docker compose -f docker-compose.yml build
-
+clean:
```

Comment:
Is this needed? Won't docker-compose handle cleanup? Also how do we know the name of the image will be 'tor-tor-*`?